sdk/appconfiguration/azure-data-appconfiguration/CHANGELOG.md

```diff
@@ -1,6 +1,7 @@
 # Release History
-## 1.2.0-beta.1 (Unreleased)
 
+## 1.1.6 (2020-10-06)
+- Update dependency version, `azure-core` to 1.9.0 and `azure-core-http-netty` to 1.6.2.
```

Comment:
Needs a title `### Dependency updates`

Reply:
thought of adding this subtitle, but since there is only one update on dependency version,  should I still need to add the subtitle?

Reply:
yes, please :)